src/sdk/python/examples/map_reduce/run.py

```diff
@@ -10,24 +10,24 @@
 
 from functools import reduce
 import random
-import os
 import asyncio
 import logging
 
+from iotea.core.protocol_gateway import ProtocolGateway
 from iotea.core.util.logger import Logger
+from iotea.core.util.mqtt_client import MqttProtocolAdapter
+
 logging.setLoggerClass(Logger)
 logging.getLogger().setLevel(logging.INFO)
 
-os.environ['MQTT_TOPIC_NS'] = 'iotea/'
-
 # pylint: disable=wrong-import-position
 from iotea.core.talent_mr import Mapper, Worker, Reducer
 from iotea.core.rules import OpConstraint, Rule
 from iotea.core.constants import VALUE_TYPE_RAW
 
 class RandomMapper(Mapper):
-    def __init__(self, connection_string):
-        super(RandomMapper, self).__init__('mapper', 'reducer', connection_string)
+    def __init__(self, protocol_gateway_config):
+        super(RandomMapper, self).__init__('mapper', 'reducer', protocol_gateway_config)
```

Comment:
I think it should be `super().__init__` (to be aligned with the other python examples)